lib/rubocop/cop/style/redundant_format.rb

```diff
@@ -12,6 +12,22 @@ module Style
       # inlined into a string easily. This applies to the `%s`, `%d`, `%i`, `%u`, and
       # `%f` format specifiers.
       #
+      # @safety
+      #   This cop's autocorrection is unsafe because string object returned by
+      #   `format` and `sprintf` are not always frozen. If `format('string')` is autocorrected to
```

Comment:
Are there frozen at all in some case?

Reply:
Oops, as far as I know, there are no cases where they are frozen. I have adjusted the text to say "never frozen".